CHANGELOG.md

```diff
@@ -520,6 +520,7 @@ By @MarijnS95 in [#6006](https://github.com/gfx-rs/wgpu/pull/6006).
 
 #### General
 
+- Added an example that splits up large data across multiple buffers and accesses them as a single contiguous array in the shader, demonstrating the naga/wgsl only `binding_array`, and some of the, less documented `wgpu::Features`. By @alphastrata in [#6138](https://github.com/gfx-rs/wgpu/pull/6138)
```

Comment:
I made a similar comment in the main review, but: this just doesn't describe what the example actually does. I feel like something like this would be better:

```suggestion
- Added an example that shows how to handle datasets too large to fit in a single `GPUBuffer` by distributing it across many buffers, and then having the shader receive them as a `binding_array` of storage buffers.  By @alphastrata in [#6138](https://github.com/gfx-rs/wgpu/pull/6138)
```

Reply:
Thanks @jimblandy, great review, lots of good catches --(totally disagree with you on the for loops :p but, it's not my repo so I'll get in line). I'll make all these changes at some stage over the next few days.

Reply:
@alphastrata If you're curious, I put my rationale for preferring ordinary control flow (like `for` loops) over combinator chains here: https://github.com/gfx-rs/wgpu/pull/6148#discussion_r1826262913